frontend/components/bc/header/MainHeader.vue

```diff
@@ -68,20 +68,25 @@ const toggleMegaMenu = (evt: Event) => {
 
 const isMobileMegaMenuOpen = computed(() => megaMenu.value?.isMobileMenuOpen)
 
-const userMenu = computed(() => {
-  return [
-    {
-      command: async () => {
-        await navigateTo('../user/settings')
-      },
-      label: $t('header.settings'),
-    },
-    {
-      command: () => doLogout(),
-      label: $t('header.logout'),
-    },
-  ]
-})
+const labelSettings = $t('header.settings')
+const labelLogout = $t('header.logout')
+type UserMenuItem = { label: string }
+const userMenu: UserMenuItem[] = [
+  {
+    label: labelSettings,
+  },
+  {
+    label: labelLogout,
+  },
+]
+const handleUserMenuSelect = async (value: UserMenuItem) => {
+  if (value.label === labelSettings) {
```

Comment:
I am not sure if this solution is salable if we have more options and also using the translation as Identifier is not the best choice I think.  

Here is a suggestions:
```suggestion
type UserMenuItem = { command?: () => (Promise<void> | void), label: string }
const userMenu: UserMenuItem[] = [
  {
    command: async () => { await navigateTo('/user/settings') },
    label: $t('header.settings'),
  },
  {
    command: () => doLogout(),
    label: $t('header.logout'),
  },
]
const handleUserMenuSelect = async (value: UserMenuItem) => {
  await value.command?.()
}
```

Reply:
I am following yr suggestion...thx